pkgs/applications/science/logic/cvc3/default.nix

```diff
@@ -13,6 +13,24 @@ stdenv.mkDerivation rec {
 
   patches = [ ./cvc3-2.4.1-gccv6-fix.patch ];
 
+  # fails to configure on darwin due to gmp not found
+  configureFlags = [ "LIBS=-L${gmp}/lib" "CXXFLAGS=-I${gmp.dev}/include" ];
+
+  # disable stackprotector on aarch64-darwin for now
+  # https://github.com/NixOS/nixpkgs/issues/127608
+  #
+  # build error:
+  #
+  # configure:3774: checking for gmp
+  # configure:3793: g++ -o conftest -I/nix/store/lwqndzvzrdnrw7gbh7q4wl1w0i72d2cv-gmp-with-cxx-6.2.1-dev/include   conftest.cpp -lgmp
+  # -L/nix/store/9flawn1k59bmvs1x3xndbg29kq72610m-gmp-with-cxx-6.2.1/lib >&5
+  # /private/tmp/nix-build-cvc3-2.4.1.drv-0/cc21OfBQ.s:13:15: error: index must be an integer in range [-256, 255].
+  #         ldr     x1, [x1, ___stack_chk_guard];momd
+  #                          ^
+  # /private/tmp/nix-build-cvc3-2.4.1.drv-0/cc21OfBQ.s:20:15: error: index must be an integer in range [-256, 255].
+  #         ldr     x0, [x0, ___stack_chk_guard];momd
```

Comment:
Not sure we need the build error in the comment. It's quite big and messy.

Reply:
Sure, removed.